packages/web3-eth/src/types.ts

```diff
@@ -0,0 +1,194 @@
+import { AccessList, Log, TransactionHash, TransactionInfo, Uncles } from 'web3-common';
+import {
+	Address,
+	HexString,
+	Numbers,
+	ValidTypes,
+	ValidReturnTypes,
+	HexString32Bytes,
+	HexString256Bytes,
+	HexStringBytes,
+	Uint,
+} from 'web3-utils';
+
+export enum ChainNames {
+	MAINNET = 'mainnet',
+	GOERLI = 'goerli',
+	KOVAN = 'kovan',
+	RINKEBY = 'rinkeby',
+	ROPSTEN = 'ropsten',
+}
+
+export enum HardForks {
+	CHAIN_START = 'chainstart',
```

Comment:
this was called Frontier

Reply:
Created an [issue](https://github.com/ChainSafe/web3.js/issues/4633) for this, and added it to [Web3Eth TODO list](https://github.com/ChainSafe/web3.js/issues/4608)